packages/runtime/src/templates/getHandler.ts

```diff
@@ -116,14 +116,6 @@ const makeHandler = (conf: NextConfig, app, pageRoot, staticManifest: Array<[str
     const query = new URLSearchParams(event.queryStringParameters).toString()
     event.path = query ? `${event.path}?${query}` : event.path
 
-    const graphToken = event.netlifyGraphToken
```

Comment:
Looks good to me @dwwoelfel. I believe this is the only place in the code we reference OneGraph.